objects_tdd/spec/lib/cash_register_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rspec'
+require_relative '../../lib/cash_register'
+
+describe CashRegister do
+  it 'should be a cashregister' do
+    expect(subject).to be_a(CashRegister)
+  end
+
+  describe '#total' do
+    it 'should return the total' do
+      expect(subject.total).to eq(0)
+    end
+  end
+
+  describe '#add' do
+    it 'should add to a running total' do
+      subject.add(3)
+      subject.add(5)
+      expect(subject.total).to eq(8)
+    end
+  end
+
+  describe '#subtract' do
+    it 'should subtract from the running total' do
+      subject.subtract(3)
+      expect(subject.total).to eq(-3)
+    end
+
+
+    describe '#pay' do
+      context 'should pay to balance' do
```

Comment:
Generally speaking, you should have one context, one it block. The context helps further specify what each it block is doing. Here, it would be much cleaner if you wrote two contexts like this:
```
describe '#pay' do
  context 'when payment is less than total' do
    it 'should do something' do
      #code
    end
  end

  context 'when payment is equal to total' do
    it 'should do something' do
      #code
    end
  end

  context 'when payment is greater than total' do
    it 'should do something' do
      #code
    end
  end
end